src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -77,29 +77,31 @@ export class AdditionalServiceModalComponent {
     private configurationControlService: ConfigurationControlService,
     private additionalServicesUtils: AdditionalServicesUtils,
     public rolesService: RolesService) {
-      this.activatedRoute.queryParams.subscribe(params => {
-        if (params?.name) {
+      this.activatedRoute.paramMap
+      .pipe(map(() => window.history.state)).subscribe(res=>{
```

Comment:
`map(() => window.history.state)` is this required?

Reply:
Yes, it is required to fetch passed object from navbar component.